ci/snap/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+# not a tagged release, abort
+[[ "$TRAVIS_TAG" != "$TRAVIS_BRANCH" ]] && exit 0
+
+openssl enc \
+  -aes-256-cbc \
+  -md sha512 \
+  -pbkdf2 \
+  -iter 1000 \
+  -a -d \
+  -in .snapcraft/travis_snapcraft.cfg \
```

Comment:
can we move this to `snap/`? We already have 2 folders for snap (`snap/` and `ci/snap/`), adding a third is pretty annoying.